nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -527,6 +527,12 @@
      features.
    </para>
   </listitem>
+  <listitem>
+   <para>
+     We no longer ship the custom font rendering settings from <option>fonts.fontconfig.penultimate.enable</option>.
```

Comment:
Could we tweak the wording here?

```suggestion
     We no longer enable custom font rendering settings with <option>fonts.fontconfig.penultimate.enable</option> by default.
```

This doesn't seem to remove support entirely, only to change the default.

Reply:
Yeah, I was wondering is the working could be misinterpreted. Will change :+1: